acceptance/bundle/artifacts/whl_via_environment_key/databricks.yml

```diff
@@ -5,7 +5,7 @@ artifacts:
     my_test_code:
       type: whl
       path: "./my_test_code"
-      build: "python3 setup.py bdist_wheel"
+      build: python setup.py bdist_wheel
```

Comment:
Is `python3` not aliased?

Reply:
There is no 'python3' in virtualenv on Windows (as we learned in https://github.com/databricks/cli/pull/2034).